app/precompiled.py

```diff
@@ -234,9 +225,54 @@ def overlay_template_areas(src_pdf, page_number=None, overlay=True):
     return png_from_pdf(pdf, page_number)
 
 
-def get_invalid_pages(src_pdf):
-    pdf_to_validate = _overlay_printable_areas(src_pdf)
-    return list(_get_out_of_bounds_pages(PdfFileReader(pdf_to_validate)))
+def get_invalid_pages_with_message(src_pdf):
+    message = ""
+    invalid_pages = []
+    are_pages_A4_portrait_oriented, invalid_pages = _are_pages_A4_portrait_oriented(src_pdf)
+    if not are_pages_A4_portrait_oriented:
+        message = 'PDF not conforming to A4 size portrait orientation on '
```

Comment:
Suggested content change: `The page orientation is landscape instead of portrait on `

Reply:
@CrystalPea is the page orientation the only thing that will trigger this message? Or is it checking that the size is A4 as well?

Reply:
@karlchillmaid for now only orientation, in the future we also plan to add size checker, but first we want to run it silently for a while to see if our users are within limits without failing them :)

Reply:
OK. If we decide to turn that on, then page size will probably need a unique validation error message too.